### DISCUSSION TITLE
French localization

### DISCUSSION THREAD

**Discussion by sosoBLK** [janstehno/cotw-castmate#4] [Ideas]
Sur l'application vous pourriez ajouter les meilleurs spot a poisson de chaque Map et de rajouter la langue française
Cordialement

> On the application you could add the best fish spots for each map and add the French language.
> Sincerely
> translated with deepl.com

**Comment by janstehno** [comment]
Other languages are planned as well. It takes time.

**Comment by janstehno** [comment]
French localization was added in the version 1.0.6.
